### ISSUE TITLE
[Feature Request] Support for an explicit MGET 

### ISSUE THREAD

**Issue by vamshiaruru-virgodesigns** [redis-rs/redis-rs#727]
Hi all, 
The current implementation of get 
```rust

    fn get<K: ToRedisArgs>(key: K) {
        cmd(if key.is_single_arg() { "GET" } else { "MGET" }).arg(key)
    }
```
Implicitly does as MGET if key is a vector or array with **more than one element**. This is a problem because many times I don't know the length of vector I am passing to the key. So I do not know if I'll get back a single response or list of responses. I believe specifically adding an MGET command that always returns a list will make things lot easier. Would it be as simple as adding
```rust
    fn mget<K: ToRedisArgs>(key: K) {
        cmd("MGET").arg(key)
    }
```
In `commands/mod.rs`?

**Comment by vamshiaruru-virgodesigns** [comment]
I'm willing to raise a PR for this, I've implemented the changes in my fork: https://github.com/vamshiaruru-virgodesigns/redis-rs

**Comment by jaymell** [comment]
Please feel free to raise a PR. The original API here is unfortunate, and this seems like a simple solution for the issues people have reported in #336 as well.

**Comment by vamshiaruru-virgodesigns** [comment]
@jaymell I have raised the PR, can you please have a look? https://github.com/redis-rs/redis-rs/pull/729

**Comment by jaymell** [comment]
Fixed by #729
